raudio_bot/__main__.py

```diff
@@ -10,6 +10,36 @@
 from dotenv import load_dotenv
 
 
+import json
+
+def check_for_file(file_name):
+    if os.path.exists(file_name):
+        return True
+    else:
+        x = input("Currently, there is no configuration specified for this bot. Would you like to create one? (y/n): ")
+
+        if (x == 'y'):
+            # get url and create data dictionary containing default specifications
+            url = input("Enter URL of server you will be streaming from: ")
+
+            # ADD ANY OTHER DEFAULT CONFIGS HERE
+            data = {'stream_url': url, 'authenticated': [ ]}
```

Comment:
Ideally we will want to interface with our `RaudioConfig` class instead, which would save us from needing to specify the config in two places. You can declare it like any other class.

```suggestion
            data = RaudioConfig(stream_url=url, authenticated=[])
```